### PR TITLE
Add image-select question type

### PR DESCRIPTION
**Pull request by yaseenshaik**
New files:
category.js - to handle category view
image-select.js - new question type with corresponding scss file
englishData.js - questions for english
chime.mp3 and buzz.mp3 - sound files and corresponding edit to webpack config.

Modified:
chapter.js and home-page.js - to accommodate any number of chapters
transitions.scss - older ones were not working, so had to update them.
styles.scss - as required.

### CODE REVIEW

File: src/js/chapter-data/englishData.js [hunk 1]
```diff
@@ -0,0 +1,218 @@
+const englishData = [{
+  // causes it to not show up on the homepage
```
Comment (nicolasartman) [inline src/js/chapter-data/englishData.js]: is that comment correct or just left over?

File: src/js/image-select.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        );
+      }
+    });
+  },
+
+  playSound(name) {
+    this[name].currentTime = 0;
+    this[name].volume = 0.6;
+    this[name].play();
+  },
```
Comment (nicolasartman) [inline src/js/image-select.js:36]: i'm not sure about browser compatibility here. are you confident this will work on older mobile browsers?

**Review by nicolasartman — COMMENTED** [review]
just a couple questions, but generally looking very good!

File: src/js/chapter-data/englishData.js (hunk 1 above)
Comment (yaseenshaik) [reply src/js/chapter-data/englishData.js]: That's probably left over from demoData.js (from the original chapter 999).

File: src/js/image-select.js (hunk 2 above)
Comment (yaseenshaik) [reply src/js/image-select.js:36]: Yeah, this should not be a problem https://caniuse.com/#search=audio

Comment (nicolasartman) [reply src/js/image-select.js:36]: interesting, so webpack is creating an audio tag and exposing its api for you to use? if so, sgtm (sounds good to me)

File: src/js/chapter-data/englishData.js (hunk 1 above)
Comment (nicolasartman) [reply src/js/chapter-data/englishData.js]: mind removing it? thanks

File: src/js/image-select.js (hunk 2 above)
Comment (yaseenshaik) [reply src/js/image-select.js:36]: Not webpack but React Refs: https://reactjs.org/docs/refs-and-the-dom
Refer: L107 & 108 in the same file

Comment (nicolasartman) [reply src/js/image-select.js:36]: ohhhhh I see what you did there. I missed where you actually just used an audio tag and kept the ref. seems alright to me.

**Pull request merged**